doozerlib/cli/__main__.py

```diff
@@ -397,8 +397,9 @@ def images_update_dockerfile(runtime, stream, version, release, repo_type, messa
 
 @cli.command("config:scan-sources", short_help="Determine if source repo differs from distgit content")
 @click.option("--yaml", "as_yaml", default=False, is_flag=True, help='Print results in a yaml block')
+@click.option("--include-disabled", default=False, is_flag=True, help="Including disabled images")
```

Comment:
There is an option like this on the runtime:
```
--load-disabled              Treat disabled images/rpms as if they were enabled
```

I think I would prefer to just reuse that here.

Reply:
Makes sense. I've changed the code to respect that option.